ocs_ci/deployment/baremetal.py

```diff
@@ -663,8 +663,8 @@ def create_pxe_files(self, ocp_version, role):
     MENU LABEL PXE Boot
     MENU DEFAULT
     KERNEL rhcos-installer-kernel-x86_64
-    APPEND ip=dhcp rd.neednet=1 initrd=rhcos-installer-initramfs.x86_64.img console=ttyS0 console=tty0 coreos.inst=yes \
-coreos.inst.install_dev=sda {bm_metal_loc} coreos.inst.ignition_url={bm_install_files_loc}{role}.ign \
+    APPEND ip=enp1s0f0:dhcp ip=enp1s0f1:none rd.neednet=1 initrd=rhcos-installer-initramfs.x86_64.img console=ttyS0 console=tty0 \
```

Comment:
tox: flake8
  flake8: commands[0]> flake8 ocs_ci tests
  ocs_ci/deployment/baremetal.py:666:121: E501 line too long (130 > 120 characters)
  flake8: exit 1 (13.31 seconds) /home/runner/work/ocs-ci/ocs-ci> flake8 ocs_ci tests pid=3657
flake8: FAIL ✖ in 16.98 seconds


You will need to fix this